scripts/ddp-RxTerms-script/src/index.ts

```diff
@@ -0,0 +1,53 @@
+import * as XLSX from "xlsx";
+import { RxNormDrugOption } from "./models";
+import { parseToHOCON, writeFile } from "./utils/common";
+
+const inputFileName = process.argv[2] || "RxTerms202202.txt";
+const outputFileName = process.argv[3] || "output.txt";
+
+const translations: string[] =  process.argv.filter(arg => ['en', 'es', 'ru'].includes(arg));
+
+const workBook = XLSX.readFile(inputFileName);
+
+const workSheet = workBook.Sheets[workBook.SheetNames[0]];
+
+const rxNormArray = XLSX.utils.sheet_to_json(workSheet) as RxNormDrugOption[];
+
+const filteredRxNormOptions = rxNormArray.map((drug) => ({
+  stableId: drug.RXCUI,
+  displayName: drug.DISPLAY_NAME
+}));
+
+const pickListOptions = filteredRxNormOptions.map((option) => ({
+  stableId: option.stableId,
+  optionLabelTemplate: {
+    templateType: "TEXT",
+    templateText: option.displayName
+  }
+}));
+
+const picklistString = pickListOptions.reduce((prev, current) => {
+  const stringBuilder = `{${Object.keys(current)[0]}: ${
+    current.stableId
+  } optionLabelTemplate:{${Object.keys(current.optionLabelTemplate)[0]}: "${
+    current.optionLabelTemplate.templateType
+  }" ${
+    Object.keys(current.optionLabelTemplate)[1]
+  }: "$drug_name"
+  variables: [
+    {
+      name: drug_name
+      translations: [
+         ${translations.map(lang => `{language:  ${lang}  text:  "${current.optionLabelTemplate.templateText.replace(/'/g, "")}"}` )}
+      ]
+    }
+  ]  
+}},`;
+  return (prev += stringBuilder);
+}, "");
+
```

Comment:
All the stuff above I don't think makes sense to do.
You are creating here a gigantic HOCON string, but then, when you write it out, you convert to an object (using HOCON parser) and THEN to JSON string. 
So complicated AND we throw the work away.

Reply:
Not really. Hocon parser library allows to provide strings without commas or quotes: 
![image](https://user-images.githubusercontent.com/98110251/162110820-dce09a11-8877-45a8-a360-085c2efaf5e9.png)
I parse it to HOCON just to double check, and add all needed symbols.
Not creating a HOCON's string particularly just a string,